controller/workitems.go

```diff
@@ -225,9 +225,14 @@ func (c *WorkitemsController) List(ctx *app.ListWorkitemsContext) error {
 	offset, limit := computePagingLimits(ctx.PageOffset, ctx.PageLimit)
 	var workitems []workitem.WorkItem
 	var count int
+	sort, err := workitem.ParseSortWorkItemsBy(ctx.Sort)
+	if err != nil {
+		return jsonapi.JSONErrorResponse(ctx, err)
```

Comment:
@baijum, shouldn't we return a `InternalServerErorr` here instead of `JSONErrorResponse`?

Reply:
`JSONErrorResponse` constructs HTTP response based on the actual underneath error. In this case, if user is providing  an unspoorted sort option, it will give back `NewBadParameterError` (400).

Reply:
Thanks @baijum